ooniprobe/View/TestResults/LogViewController.m

```diff
@@ -20,11 +23,14 @@ - (void)viewDidLoad {
         NSString *fileName = [self.measurement getLogFile];
         NSString *content = [TestUtility getUTF8FileContent:fileName];
         if (content != nil) {
-            [self.textView setText:content];
+            self.text = content;
+            [self.webView loadData:[content dataUsingEncoding:NSUTF8StringEncoding]
```

Comment:
What happens is the data is not UTF-8? (It should be but that is not enforced on the language level.)

Reply:
I guess will encode/show the data in a wrong way